### ISSUE TITLE
Custom style css selector

### ISSUE THREAD

**Issue by zagortenay333** [collinss/cinnamon-places-center#18]
Can you please update the function that adds the custom style class to this applet?

**Comment by collinss** [comment]
Sorry, I just moved and don't have access to the Internet at home atm. I'll try to get to this when I can, but it might be a while.

**Comment by zagortenay333** [comment]
It's just a one liner.. Want me to make a pull?

**Comment by zagortenay333** [comment]
Granted I don't know how you deal with backward portability.

**Comment by collinss** [comment]
Yeah, backward compatibility is probably a little bit more work than a one-liner. I think I also had some other things to update on this as well, but I can't recall for sure right now. You could submit a pull request, but it might still be a while before an updated version makes it on to the spices website.

**Comment by zagortenay333** [comment]
Np, then. I can wait :smile:
